### ISSUE TITLE
Tests are very noisy

### ISSUE THREAD

**Issue by jackfirth** [samth/docker-racket-build#27]
Running all the release tests in the `test.sh` and `test-gui.sh` scripts results in well over ten thousand lines of stdout output, which is Travis CI's limit for how much output will be displayed in the browser. We should explore which tests are the noisiest and most easily silenced.

**Comment by samth** [comment]
A first one to address is the htdp tests, which can be made quieter using `tests/racket/quiet.rktl`.

**Comment by jackfirth** [comment]
Instead of silencing existing output using wrappers and without changing the tests, I'd prefer to change the tests to simply not show unnecessary output in the first place when run in their default setup. How does that `quiet.rktl` script work exactly?

**Comment by samth** [comment]
I don't think we want to change people's automated tests for them, and instead we should either build variations or pass options on the command line (such as `-Q` to `raco test`).

**Comment by jackfirth** [comment]
Variations and options means the same tests are run differently for releases than they're run during routine development. I'd like to minimize that as much as possible because otherwise tests may pass in one setup and fail in another without it being clear which way to run the tests is "correct". This bit me with the Travis CI setup in rackunit, as the Travis script didn't use `--drdr` and thought stderr output was okay while DrDr noticed that some tests started producing stderr output unexpectedly.

I don't think we need an immediate short term fix for this, since large Travis logs are only an inconvenience.
